extension/chrome/elements/attachment_preview_pdf.ts

```diff
@@ -0,0 +1,141 @@
+/* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
```

Comment:
Sorry that we don't have an obvious place to put these. I'm thinking this file could go in https://github.com/FlowCrypt/flowcrypt-browser/tree/master/extension/js/common/ui/ - it's not a view file, and it's UI related, so I think the UI folder will be a good home for it.

Reply:
Done :+1: